lab_1/update-pipeline-definition.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash -
+#===============================================================================
+#
+#          FILE: update-pipeline-definition.sh
+#
+#         USAGE: ./update-pipeline-definition.sh
+#
+#   DESCRIPTION: 
+#
+#       OPTIONS: ---
+#  REQUIREMENTS: ---
+#          BUGS: ---
+#         NOTES: ---
+#        AUTHOR: YOUR NAME (), 
+#  ORGANIZATION: 
+#       CREATED: 03/15/2023 10:12:08 PM
+#      REVISION:  ---
+#===============================================================================
+
+set -o nounset                                  # Treat unset variables as an error
+
+checkJQ() {
+  # jq test
+  type jq >/dev/null 2>&1
+  exitCode=$?
+	jqDependency=1
```

Comment:
do we really need this jqDependency as  we already have "type" command run result in exitCode?

Reply:
fixed, thanks